my-app/src/app/modules/courses-page/courses-page.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="form-container">
+  <form>
+    <div class="input-container">
+      <i class="fa-solid fa-magnifying-glass search-icon"></i>
+      <input type="text"
+             [(ngModel)]="searchValue"
+             name="courseName"
+             placeholder="Input a course name"
+             class="search-courses-input">
+    </div>
+    <button class="search-button" (click)="onSearchCourse(searchValue)">Search</button>
+  </form>
+</div>
+<div class="add-course-button-container">
+  <button class="add-course-button">
+    <i class="fa-solid fa-plus"></i>
+    <p class="add-course-button-text">Add course</p>
+  </button>
+</div>
+<ng-container *ngIf="courses && courses.length !== 0">
+  <app-courses
+    *ngFor="let iterCourse of courses | orderBy: 'creationDate'; trackBy: trackByFn"
+    [course]="iterCourse"
+    (deleteCourse)="onDeleteCourse($event)"
+  >
+  </app-courses>
+  <app-load-more-button></app-load-more-button>
+</ng-container>
+<div class="no-data-container" *ngIf="!courses || courses.length === 0">
+  <h3>No data. Feel free to add new course.</h3>
+</div>
```

Comment:
I see search, course adding, and displaying a full list of courses here. Too many responsibilities for one component, as for me.

Reply:
At least separate them visually with newlines

Reply:
Added lines, but let's discuss on our meeting too, please